fighthealthinsurance/views.py

```
@@ -147,6 +147,65 @@ class OtherResourcesView(generic.TemplateView):
+            os.path.dirname(os.path.abspath(__file__)),
+            "static", "blog_posts.json"
+        )
+        post_info = {}
+        try:
+            with open(blog_json_path, "r", encoding="utf-8") as f:
+                posts = json.load(f)
+            for post in posts:
+                if post.get("slug") == slug:
+                    post_info = post
+                    break
+        except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)